src/modules/Authorization/Authorization.tsx

```diff
@@ -1,201 +1,112 @@
-/* eslint-disable */
-// @ts-nocheck
-import { Link } from 'react-router-dom';
 import './Authorization.scss';
 import { useState } from 'react';
-import imageCard from '../../assets/img/author.png';
-import Header from '../../components/Header/Header';
-import CommonButton from '../../UI/CommonButton/CommonButton';
-// import PropTypes from 'prop-types';
-import { Footer } from '../../components/Footer/Footer';
+import { Link, useNavigate } from 'react-router-dom';
 import { useAppDispatch } from '../../redux/store';
-import { authentication } from '../auth/authReducer';
+import { updateUser } from '../../redux/reducers/userSlice';
+import { ChangeInputEvent, SubmitFormEvent } from '../../@types/types';
 import { Path } from '../../pages/constants/path';
-import EyeIcon from '../../assets/icons/IconsSvg/EyeIcon';
-import EyeIconHidden from '../../assets/icons/IconsSvg/EyeIconHidden';
 import MainApiService from '../../api/MainApiService';
+import AddLinks from '../../components/AddLinks/AddLinks';
+import CommonButton from '../../UI/CommonButton/CommonButton';
+import { validMail, validPassword } from '../../helpers';
 
-export const Authorization = ({ primary }) => {
-  const [passwordVisible, setPasswordVisible] = useState(false);
+export default function UserAuthorization() {
   const [email, setEmail] = useState('');
   const [password, setPassword] = useState('');
-  const [emailError, setEmailError] = useState('');
-  const [passwordError, setPasswordError] = useState('');
+  // const [passwordVisible, setPasswordVisible] = useState(false);
   const [rememberMe, setRememberMe] = useState(false);
   const dispatch = useAppDispatch();
-  const [emailInputText, setEmailInputText] = useState('');
-  const [passwordInputText, setPasswordInputText] = useState('');
 
-  const handleRememberMeChange = (event) => {
-    setRememberMe(event.target.checked);
-  };
+  const navigate = useNavigate();
 
-  const errorMessages = {
-    required: 'Это поле обязательно для заполнения',
-    invalid: 'Неверный логин или пароль',
+  const handleRememberMeChange = () => {
+    setRememberMe(!rememberMe);
   };
 
-  const handleSubmit = async (event) => {
+  // const errorMessages = {
+  //   required: 'Это поле обязательно для заполнения',
+  //   invalid: 'Неверный логин или пароль',
+  // };
```

Comment:
Add TODO in comment if this part of code should be saved or used later

Reply:
Fix this